macros/blueprint-proc-macro-playground/src/lib.rs

```diff
@@ -20,12 +20,12 @@ impl std::fmt::Display for Error {
 
 impl std::error::Error for Error {}
 
-struct MyContext;
+pub struct MyContext;
```

Comment:
This is some start but I'm curious what your plans are here and if we are on the same page?

I expect the context to not be something created by the developer but rather something created by us at the SDK level. It should have a config that we can plug in things related to the Environment (or multiple Environments).